### DISCUSSION TITLE
Pull Shark badge not updating!

### DISCUSSION THREAD

**Discussion by cyai** [community/community#18755] [Other Feature Feedback, Questions, & Ideas] [answered]
I have opened many pull requests and many of them have been merged. But still, I can see only '2x' Pull shark badge on my profile! 

It is mentioned to be a beta version, but I think the number of PR of mine merged should be counted!

I am not able to know how the badge counting works, can someone guide me?

**Comment by JonasJs** [comment, marked as answer]
The answer is in this link https://github.com/orgs/github-community/discussions/18156 😄

**Reply by cyai** [reply]
Thank you, I got it! 😊
